flaml/automl.py

```diff
@@ -2022,11 +2028,24 @@ def custom_metric(
             import ray
 
             n_cpus = use_ray and ray.available_resources()["CPU"] or os.cpu_count()
+            if ray.available_resources().get("GPU"):
+                n_concurrent_trials = int(
+                    ray.available_resources().get("GPU")
+                    if use_ray
+                    else n_concurrent_trials
+                )
```

Comment:
When n_concurrent_trials is specified, this statement will override the user-specified value. Why?